Tasks/AzureAppServiceManage/extensionmanage.ts

```diff
@@ -64,19 +65,48 @@ export async function installExtension(publishingProfile, extension: string) {
     return defer.promise;
 }
 
-export async function installExtensions(publishingProfile, extensions: Array<string>) {
+export async function installExtensions(publishingProfile, extensions: Array<string>, extensionOutputVariables: Array<string>) {
+
+    var outputVariableCount = 0;
+    var outputVariableSize = extensionOutputVariables.length;
     var InstalledExtensions = await getInstalledExtensions(publishingProfile);
+    var extensionInfo = null;
     var anyExtensionInstalled = false;
     for(var extension of extensions) {
         extension = extension.trim();
         if(InstalledExtensions[extension]) {
-            console.log(tl.loc('ExtensionAlreadyAvaiable', InstalledExtensions[extension]));
+            extensionInfo = InstalledExtensions[extension];
+            console.log(tl.loc('ExtensionAlreadyAvaiable', extensionInfo['title']));
         }
         else {
             tl.debug("Extension '" + extension + "' not installed. Installing...");
-            await installExtension(publishingProfile, extension);
+            extensionInfo = await installExtension(publishingProfile, extension);
             anyExtensionInstalled = true;
         }
+        if(outputVariableCount < outputVariableSize) {
+            var extensionLocalPath: string = getExtensionLocalPath(extensionInfo);
+            tl.debug('Set Variable ' + extensionOutputVariables[outputVariableCount] + ' to value: ' + extensionLocalPath);
+            tl.setVariable(extensionOutputVariables[outputVariableCount], extensionLocalPath);
+            outputVariableCount += 1;
+        }
     }
     return anyExtensionInstalled;
 }
+
+function getExtensionLocalPath(extensionInfo: JSON): string {
+    var extensionId: string = extensionInfo['id'];
+    var homeDir = "D:\\home\\";
+
+    if(extensionId.startsWith('python2')) {
+        return homeDir + "Python27";
+    }
+    else if(extensionId.startsWith('python351') || extensionId.startsWith('python352')) {
```

Comment:
this may not be required as we are not showing option to install 3.5 < python version  < 3.5.3 ?